src/scheduler/node_info.c

```diff
@@ -3912,6 +3952,64 @@ parse_selspec(char *select_spec)
 }
 
 /**
+ *	@brief compare two chunks for equality
+ *	@param[in] c1 - first chunk
+ * 	@param[in] c2 - second chunk
+ *
+ *	@return int
+ *	@retval 1 if chunks are equal
+ *	@retval 0 if chunks are not equal
+ */
+int compare_chunk(chunk *c1, chunk *c2) {
+	if (c1 == NULL && c2 == NULL)
+		return 1;
+	if (c1 == NULL || c2 == NULL)
+		return 0;
+	
+	if(strcmp(c1->str_chunk, c2->str_chunk) != 0)
```

Comment:
Good point.  I removed the strcmp() of the original string.  You are right that it's redundant.  I chose to go this way because we don't know if there will be any data derived and stored in the chunk.  Right now it is just the sequence number, but maybe in the future there could be more.